hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.tsx

```diff
@@ -82,6 +82,21 @@ describe("<PendingTable />", () => {
             expect(queryByText(`Order #${id}`)).toBeNull();
         });
     });
+
+    it("PendingTable hides cancel order button when hide all button is enabled", async () => {
+        const store = makeStoreWithEntities({
+            pendingOrders: mockPendingOrdersInTable,
+        });
+        const { getByText, queryByText } = render(<PendingTable />, {
+            store,
+        });
+        const button = getByText(/hide all/i);
+        fireEvent.click(button);
+        expect(getByText(/show all/i)).toBeInTheDocument();
+        mockPendingOrders.map(() => {
```

Comment:
if you want to check that cancel order button is not there for any order, you don't need this map because you're not using anything from the array. Instead what this code is doing is running `expect(queryByText(/cancel order/i)).toBeNull();` multiple times, which is unnecessary.